### DISCUSSION TITLE
How can I display query result when I use "Execute commands from file" in another word : `\i`?

### DISCUSSION THREAD

**Discussion by stephane-klein** [dbcli/pgcli#1460] [Q&A]
Hi,

When I execute, for instance:

```
postgres@127:postgres> SELECT 1;
+----------+
| ?column? |
|----------|
| 1        |
+----------+
SELECT 1
Time: 0.009s
```

The result 1 is displayed.

Now I put `SELECT 1;` in `playground.sql` and I execute:

```
postgres@127:postgres> \i playground.sql
Time: 0.009s
```

I don't see the `1` result.

Question: How can I configure `pgcli` to display the query results in the output (stdout) when I use the [Execute commands from file](https://www.pgcli.com/commands) feature `\i`?

Best regards,
Stéphane

**Comment by j-bennet** [comment]
Hi @stephane-klein,

Looks like the command right now doesn't produce any output, but I'm pretty sure when it was originally implemented, the intent was to output everything back to the user:

https://github.com/dbcli/pgcli/pull/334

So I think that at some point it broke. Feel free to file a bug.

Thank you!
